propel-feat.yaml

```diff
@@ -31,27 +30,40 @@ services:
       - name: HYDRA_ADMIN_URL
       - name: HYDRA_OAUTH2_INTROSPECT_URL
       - name: SKIP_FIXTURES
-- name: storefront
-  certificate_arn: arn:aws:acm:us-west-2:773713188930:certificate/c2979a7a-7b84-43ed-b3a7-24b0256a1b9f
+      - name: CERTIFICATE_ARN
+- name: storefront-core
   root_domain: staging.reactioncommerce.com
   dns_name: storefront
   desired_task_count: 1
   min_task_count: 0
-  max_task_count: 8
+  max_task_count: 2
   min_healthy_percent: 100
   max_percent: 200
   alb_listener_port: 80
   alb_listener_path: /
-  alb_health_check_path: /
+  alb_health_check_path: /health
   task-definition:
-    name: staging-storefront
+    name: staging-storefront-core
     containers:
+    - name: nginx
+      cpu: 128
+      memory: 256
+      port_mappings:
+      - container_port: 8082
+        host_port: 8082
+      image: sportsdirect/nginx
```

Comment:
Is `sportsdirect/nginx` correct for this repo?

Reply:
Good catch, I fixed it.